http_parser.c

```diff
@@ -2095,6 +2095,16 @@ http_method_str (enum http_method m)
   return ELEM_AT(method_strings, m, "<unknown>");
 }
 
+const char *
+http_status_str( enum http_status s )
```

Comment:
Style: `http_status_str (enum http_status s)` - note the blanks.

Ditto on line 2103 but no blank before the `(`.